### DISCUSSION TITLE
Version 0.7

### DISCUSSION THREAD

**Discussion by woosaaahh** [woosaaahh/sj.nvim#24] [Ideas]
The coming week, I'll create the `0.7` branch. 

Here is what I'm planning to do for this branch : 

- a big refactor for the search and the highlights (shorter and more independent functions) ;
- feat: "multi windows" (a good version this time without flickering) ;
- feat: "multi highlights".

The "multi highlights" feature will only highlight the matches for a bunch of patterns with different highlights for each patterns without labels or jumps on matches, only highlights.
_e.g._ one highlight for all matches of 'foo', one highlight for all matches of 'bar' and so on, every highlight visible at the same time like you would use highlighters on a document.

:question: @cpea2506 @jemag @ziontee113
By the way, does anyone had issues with the number of available labels ? (not enough labels)
If so, was it often and was it really annoying ?

**Comment by ziontee113** [comment]
I'm still confused about the highlight feature. Will it help in jumping or just for highlighting like `nvim-colorizer`?

**Reply by woosaaahh** [reply]
No jumping only highlighting.

[`nvim-colorizer`](https://github.com/norcalli/nvim-colorizer.lua) highlights color patterns but SJ would highlight any patterns, on demand not automatically.

But you're right, a plugin might already exist for this !
